dist/tools/doccheck/exclude_simple

```diff
@@ -8424,3 +8424,4 @@ warning: Member LSM6DSXX_PARAM_GYRO_FIFO_DEC (macro definition) of file lsm6dsxx
 warning: Member LSM6DSXX_PARAMS (macro definition) of file lsm6dsxx_params.h is not documented.
 warning: Member LSM6DSXX_SAUL_INFO (macro definition) of file lsm6dsxx_params.h is not documented.
 RIOTDoxygenLayout.xml:8: warning: the type 'topics' is not supported for the entry tag within a navindex! Check your layout file!
+Error: Member AT6561_STBY_PIN (macro definition) of file board.h is not documented.
```

Comment:
```suggestion
warning: Member AT6561_STBY_PIN (macro definition) of file board.h is not documented.

```

Reply:
Please do this -- squash it right in the commit

Reply:
Done